packages/dashboard-frontend/src/containers/Loader/AbstractStep.tsx

```diff
@@ -61,7 +65,10 @@ export abstract class AbstractLoaderStep<
     const currentStep = loaderSteps.get(currentStepIndex).value;
 
     currentStep.hasError = true;
-    const lastError = common.helpers.errors.getMessage(e);
+    const lastError = {
```

Comment:
@dkwon17 As for me, I prefer to have the lastError as a simple error or as a message. Do you have a reason to have both?

@akurinnoy WDYT?

Reply:
On second thought, it looks like I don't need both. Is there a preference as to which to keep, which to remove?

If lastError is a message (string), then we don't need [RunningWorkspaceesExceededError](https://github.com/eclipse-che/che-dashboard/blob/53fc942af03fbbc11be8a08de5f1707cb8bfe162/packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts#L56-L61), and can instead do a string comparison for the error message [here](https://github.com/eclipse-che/che-dashboard/blob/53fc942af03fbbc11be8a08de5f1707cb8bfe162/packages/dashboard-frontend/src/pages/Loader/Workspace/index.tsx#L67)

Reply:
In my latest push, I have `lastError` as a simple error (`unknown` type) for now:
https://github.com/eclipse-che/che-dashboard/blob/7aecab30a8fcf108bf959e9471c7860a0527a103/packages/dashboard-frontend/src/containers/Loader/AbstractStep.tsx#L26